target/linux/ath79/dts/ar7241.dtsi

```diff
@@ -78,7 +78,7 @@
 };
 
 &eth1 {
-	compatible = "qca,ar7241-eth", "syscon", "simple-mfd";
```

Comment:
@981213 shouldn't you remove this as well? https://github.com/openwrt/openwrt/blob/71ab2c9d179652d9b528817ec04d3f1dd3b4b1df/target/linux/ath79/files/drivers/net/ethernet/atheros/ag71xx/ag71xx_main.c#L1516-L1520

Reply:
This should be kept here because it's now the only way mdio driver get probed. It can be dropped after actually merging mdio and eth driver back together.

Reply:
I mean, this in comment is no longer true or is it ? https://github.com/openwrt/openwrt/blob/71ab2c9d179652d9b528817ec04d3f1dd3b4b1df/target/linux/ath79/files/drivers/net/ethernet/atheros/ag71xx/ag71xx_main.c#L1512

Reply:
It's still true as I didn't drop the ability to use simple-mfd to probe mdio before eth.
As it'll be dropped later anyway I think it isn't necessary to use a commit only to drop/modify some comment lines.
Checking OF_POPULATED_BUS is still necessary because the driver doesn't unregister the mdio node after probe deferring. It should be kept just to prevent double registering of mdio device.